lib/common.js

```diff
@@ -161,13 +161,21 @@ var restoreOverriddenRequests = function() {
   requestOverride = [];
 };
 
+/**
+ * Get high level information about request as string
+ * @param  {Object} options
+ * @param  {string} options.method
+ * @param  {string} options.port
+ * @param  {string} options.proto
+ * @param  {string} options.hostname
+ * @param  {string} options.path
+ * @param  {Object} options.headers
+ * @param  {string|object} body
```

Comment:
@anotheredward The query string will be included in the path option. The reference should be as well. You can see this is validated within the tests: https://github.com/node-nock/nock/pull/597/files#diff-ae3b1775e1016e4912805bd8eb567550R4920